src/Phinx/Migration/AbstractMigration.php

```diff
@@ -340,9 +340,18 @@ public function dropTable($tableName)
      */
     public function preFlightCheck($direction = null)
     {
-        if (method_exists($this, MigrationInterface::CHANGE)) {
-            if (method_exists($this, MigrationInterface::UP) ||
-                method_exists($this, MigrationInterface::DOWN) ) {
+        $abstractBaseClassname = 'Phinx\Migration\AbstractMigration';
+        $refl   = new \ReflectionClass($this);
```

Comment:
As Phinx is now PHP 5.6+, you can use `Phinx\Migration\AbstractMigration::class` rather than a string reference `'Phinx\Migration\AbstractMigration'`.